Enrich_methods/example.py

```diff
@@ -0,0 +1,210 @@
+import argparse
+from pyne import material
+from pyne.material import Material, MultiMaterial
+from pyne.material_library import MaterialLibrary
+from material_db_tools import mix_by_volume
+import material_db_tools as mdbt
+from material_library import (
+    mat_data as pure,
+)  # this is importing the material dictionaries
+
+mat_lib_test = MaterialLibrary()
+# you may need to adjust this path for your setup
+mat_lib_test.from_json("material_library.json")
+
+
+simulation_lib = MaterialLibrary()
+
+
+################################### Method 3
+
+
+# Currently how Edgar and I are enriching out materials
+def enriched_lithium(enrichment):
+    enriched_li = Material({"Li6": enrichment, "Li7": 1 - enrichment})
+    return enriched_li
+
+
+def Li2TiO3Li_mat(enrichment=0.60):
+    Li2TiO3Li60 = Material()
+    Li2TiO3Li60.from_atom_frac(
+        {enriched_lithium(enrichment): 2, 80000000: 3, 220000000: 1}
+    )
+    Li2TiO3Li60.density = 3.43
+    Li2TiO3Li60 = Li2TiO3Li60.expand_elements()
+    Li2TiO3Li60.metadata["citation"] = "HernandezFusEngDes_2018"
+    return Li2TiO3Li60
+
+
+def Li4SiO4Li_mat(enrichment=0.60):
+    Li2TiO3Li60 = Material()
+    Li2TiO3Li60.from_atom_frac(
+        {enriched_lithium(enrichment): 4, 80000000: 4, 140000000: 1}
+    )
+    Li2TiO3Li60.density = 2.40
+    Li2TiO3Li60 = Li2TiO3Li60.expand_elements()
+    Li2TiO3Li60.metadata["citation"] = "HernandezFusEngDes_2018"
+    return Li2TiO3Li60
+
+
+# I also need to have another version of this function for the pin breeder zone.
+# I will if need to either make a version of this for each material I'm testing
```

Comment:
I will Need to either make a version of this function for each material I'm testing. Or create a universal function that will require me to pass all the arguments already in hcpb_pin_bw() along with all 8 different materials being evaluated for HCPB.